src/v0/destinations/lemnisk/utils.js

```diff
@@ -0,0 +1,19 @@
+const { TransformationError } = require("../../util/errorTypes");
+
+/**
+ * Fetches the platform type from the destination Config
+ * @param {*} destination
+ * @returns platform type used
+ */
+const fetchPlatform = destination => {
+  const { cloudMode } = destination.Config;
+  if (cloudMode === "web") {
+    return "pl";
+  }
+  if (cloudMode === "server") {
+    return "diapi";
+  }
+  throw new TransformationError("Payload contains invalid configuration");
```

Comment:
By the error message, it seems to be more inclined to config error

Reply:
Yes changed

Reply:
Done